skiko/src/commonMain/kotlin/org/jetbrains/skiko/SkikoView.kt

```diff
@@ -6,7 +6,10 @@ interface SkikoView {
     // Input
     fun onKeyboardEvent(event: SkikoKeyboardEvent) = Unit
     fun onPointerEvent(event: SkikoPointerEvent) = Unit
+
+    @Deprecated("This method will be removed. Use override val input:PlatformInput")
     fun onInputEvent(event: SkikoInputEvent) = Unit
+    val input: SkikoInput
```

Comment:
Breaking changes here

Reply:
Backward compatibility for compilation was not changed, because we provide default val input: SkikoInput = get() Empty